src/Java/Lesson5/Main.java

```diff
@@ -0,0 +1,35 @@
+package Java.Lesson5;
+
+public class Main {
+    public static void main(String[] args) {
+        PassengerAuto PassengerAuto = new PassengerAuto();
+        PassengerAuto.traffic();
+        PassengerAuto.brand();
+        PassengerAuto.Opisanie();
+        PassengerAuto.Peres4et();
+        PassengerAuto.getMaxSpead();
+
+        Truck truck = new Truck();
+        truck.trafficMaz();
+        truck.bradTruck();
+        truck.Opisanie();
+        Truck.gruz();
+
+        CivilAviation civilAviation = new CivilAviation();
+        civilAviation.trafficAir();
```

Comment:
@Ihar-Malinouski при изменении метода на возвращающий значение
```suggestion
        civilAviation.incomingPassengers<в твоем варианте на данный момент - `passengers ` = trafficAir();
```